module-cache.ts

```diff
@@ -0,0 +1,6 @@
+import {Browser} from './browser';
+export class ModuleCache {
+    constructor(browser: Browser) { var dictionary = {}; }
```

Comment:
The class should have a property for the cache. Above the constructor, put `cache: any = {};`. Remove the body of the constructor so that the constructor is `constructor(...) { }`.